game/static/game/css/board.css

```
@@ -395,10 +339,15 @@ body {
     width: 100%;
     max-width: 100%;
     margin: 0 auto;
+
+    overflow-x: hidden;
```

Stylelint (17.14.0)

[error] Duplicate property "overflow-x" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)